app/views/movies/_movie_partial_loop.html.erb

```diff
@@ -1,28 +1,27 @@
+<div class="row tile-container">
```

Comment:
This looks like a lot of changes, but it's just blankspaces. The main change is switching `@movies` to `movies` on line 2.